src/outputs.jl

```diff
@@ -9,7 +9,7 @@ Base.@kwdef mutable struct RABBIToutput
     time_data::Union{Vector{<:Real},Missing} = missing 
 end
 
-function read_outputs(path::String; filename::String ="run")
+function read_outputs(path::String, filename::String)
```

Comment:
i am surprised this would work. Are uou sure it still makes sense to join path and filename. shouldn't this just be folder?